html/ops/index.php

```diff
@@ -106,6 +106,7 @@
     <ul>
         <li><a href=\"manage_apps.php\">Manage applications</a></li>
         <li><a href=\"manage_app_versions.php\">Manage application versions</a></li>
+        <li><a href=\"manage_consent_types.php\">Manage Consent types</a></li>
```

Comment:
can you make the C in Consent lower case to match the other ones in this list?

Reply:
Fixed in e4df047